pkg/api/defaults-kubelet.go

```diff
@@ -80,7 +79,6 @@ func (cs *ContainerService) setKubeletConfig() {
 		"--image-pull-progress-deadline":      "30m",
 		"--enforce-node-allocatable":          "pods",
 		"--streaming-connection-idle-timeout": "5m",
-		"--protect-kernel-defaults":           "true",
```

Comment:
moved to defaults.go since we only want to enable this for VHD distros, this needs to happen after the distro defaults